addons/mass_mailing/static/src/iframe/mass_mailing_iframe.js

```diff
@@ -224,6 +229,20 @@ export class MassMailingIframe extends Component {
         this.state.ready = true;
     }
 
+    checkAllInline(el) {
```

Comment:
Same as in 17, we should probably rename this:
`fixInlineDynamicPlaceholders`, include the loop over `t` elements in that function, and make `checkAllInline` a local subfunction.